.eslintrc

```diff
@@ -23,6 +23,18 @@
         "argsIgnorePattern": "^_",
         "varsIgnorePattern": "^_"
       }
-    ]
+    ],
+    "import/no-unresolved": "error"
```

Comment:
Any reason not to add: https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md?

Reply:
I'd go over the the list of rules in this plugin and figure out which ones are important.
Let's also increase the severity to error for all of the rules that are set to warn right now.